vrs/utils/PixelFrame.h

```diff
@@ -276,6 +282,8 @@ class PixelFrame {
       const;
 
  private:
+  /// Helper that abstracts common logic when reading jpgs from files or from memory
+  bool readJpegFrameHelper(struct jpeg_decompress_struct cinfo, bool decodePixels = true);
```

Comment:
when you create the local static function, please pass the frame and the struct by reference.
```suggestion
  bool readJpegFrameHelper(PixelFrame& frame, struct jpeg_decompress_struct& cinfo, bool decodePixels = true);
```